include/wx/generic/private/markuptext.h

```diff
@@ -120,7 +120,7 @@ class WXDLLIMPEXP_CORE wxItemMarkupText : public wxMarkupTextBase
     // instead of generic wxDC::DrawLabel(), so is more suitable for use in
     // controls that already use DrawItemText() for its items.
     //
-    // The meaning of the flags here is different than in wxMarkupText too:
+    // The meaning of the flags here is different from in wxMarkupText too:
```

Comment:
We can't have "from in" and I don't see anything wrong with the original "than"?

Reply:
Yeah, guess this one's OK.